src/main/resources/archetype-resources/src/main/assembly/dist/bin/__artifactId__

```diff
@@ -3,5 +3,14 @@
 SCRIPTNAME=${symbol_dollar}(basename ${symbol_dollar}0)
 BINPATH=${symbol_dollar}(command readlink -f ${symbol_dollar}0 2> /dev/null || command grealpath ${symbol_dollar}0 2> /dev/null)
 APPHOME=${symbol_dollar}(dirname  ${symbol_dollar}(dirname ${symbol_dollar}BINPATH))
+CONFIG_PATH=/etc/opt/dans.knaw.nl/${symbol_dollar}SCRIPTNAME/config.yml
 
-java -jar ${symbol_dollar}APPHOME/bin/${symbol_dollar}SCRIPTNAME.jar "${symbol_dollar}@"
+# If last_arg arg is config.yml, use that instead of the one in etc
+# See: https://stackoverflow.com/a/1853993
+for last_arg; do true; done
+if [[ $(basename ${symbol_dollar}last_arg) == config.yml ]]; then
+ echo "${symbol_dollar}last_arg"
+ CONFIG_PATH=""
+fi
+
```

Comment:
@jo-pol : there is a problem with this approach if you use an option as a last argument. Improved version:
```bash
# If last_arg arg is config.yml, use that instead of the one in etc
# See: https://stackoverflow.com/a/1853993
# Note: last_arg must not start with a dash, otherwise basename will choke on it
for last_arg; do true; done
if [[ $last_arg =~ ^[^-].*$ ]] && [[ $(basename $last_arg) ==  config.yml ]]; then
  CONFIG_PATH=""
fi
```